exosphere-shared/example-apps/shared-dependency/web-server/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "test-web-server",
+  "version": "0.0.0",
+  "author": "Kevin Goslar",
+  "dependencies": {
+    "exorelay": "0.22.1",
+    "livescript": "1.5.0"
+  },
+  "description": "Example app that is running",
+  "devDependencies": {
+    "o-tools": "0.8.47"
+  },
+  "main": "server.ls",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "license": "ISC"
+}
```

Comment:
Let's remove all dependencies and scripts if they're not being used, and we should update the description to reflect the fact that this app isn't being run.